x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_diff_tab_react_diff_viewer_continued.tsx

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useContext } from 'react';
+import ReactDiffViewer, { DiffMethod } from 'react-diff-viewer-continued';
+import { get } from 'lodash';
+import {
+  EuiSpacer,
+  EuiAccordion,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiSwitch,
+  EuiHorizontalRule,
+  EuiRadioGroup,
+  useGeneratedHtmlId,
+  useEuiTheme,
+} from '@elastic/eui';
+import type { RuleFieldsDiff } from '../../../../../common/api/detection_engine/prebuilt_rules/model/diff/rule_diff/rule_diff';
+import type { RuleResponse } from '../../../../../common/api/detection_engine/model/rule_schema/rule_schemas.gen';
+
+const HIDDEN_FIELDS = ['meta', 'rule_schedule', 'version'];
```

Comment:
I know it's not relevant for the POC, but curious why rule_schedule is hidden here? The rule schedule can be updated by the TRADE team (and later on customzied by the user)